<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gscalbiPrimaryKey</class>
 <widget class="QWidget" name="gscalbiPrimaryKey">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>230</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>230</width>
    <height>230</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>230</width>
    <height>230</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>GSCALBI v1.0</string>
  </property>
  <widget class="QGroupBox" name="gbxCapas">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Agregar campo a una capa</string>
   </property>
   <widget class="QLabel" name="lblcapas">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Capas</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbCapa">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonNC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nuevo Campo</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonSC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Serializar Campo</string>
    </property>
   </widget>
   <widget class="QWidget" name="wgtNC" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <widget class="QLabel" name="lblcapas_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nuevo campo</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNuevoCampo">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>10</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="wgtSC" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <widget class="QLabel" name="lblcapas_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Serializar campo</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbAtributo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>147</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAceptar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancelar</sender>
   <signal>clicked()</signal>
   <receiver>gscalbiPrimaryKey</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
